tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2993,8 +2993,39 @@ public function userSendsPropfindRequestToSpace(string $user, string $spaceName)
 	 * @throws GuzzleException
 	 * @throws JsonException
 	 */
-	public function theResponseShouldContain(string $user, string $method, string $space, TableNode $table): void {
+	public function theResponseShouldContainSpace(string $method, string $space, TableNode $table): void {
 		$this->featureContext->verifyTableNodeColumns($table, ['key', 'value']);
+		$this->theResponseShouldContain($method, $this->getSpaceCreator($space), $space, $table);
+	}
+
+	/**
+	 * @Then /^the "([^"]*)" response to user "([^"]*)" should contain a mountpoint "([^"]*)" with these key and value pairs:$/
+	 *
+	 * @param string $method # method should be either PROPFIND or REPORT
+	 * @param string $user
+	 * @param string $mountPoint
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function theResponseShouldContainMountPoint(string $method, string $user, string $mountPoint, TableNode $table): void {
+		$this->featureContext->verifyTableNodeColumns($table, ['key', 'value']);
+		$this->theResponseShouldContain($method, $user, $mountPoint, $table);
+	}
+
+	/**
+	 * @param string $method # method should be either PROPFIND or REPORT
+	 * @param string $user
+	 * @param string $mountPoint # an entity inside a space, or the space itself
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function theResponseShouldContain(string $method, string $user, string $mountPoint, TableNode $table): void {
```

Comment:
I like it. Thanks for refactoring. Perhaps it should be `$spaceName` here instead of `$mountPoint`?

Reply:
I am not sure how to name it. It can be a space name, or a folder name that has been shared from inside the space to the user.

Reply:
Okay, it's not important thing.

Reply:
I called it `$spaceNameOrMountPoint`